README.md

```diff
@@ -22,8 +22,8 @@ secure, and production-grade cloud infrastructure.
 - [Getting Started](#getting-started)
 - [Module Argument Reference](#module-argument-reference)
   - [Top-level Arguments](#top-level-arguments)
-    - [Module Configuration](#module-configuration)
     - [Main Resource Configuration](#main-resource-configuration)
+    - [Module Configuration](#module-configuration)
     - [Extended Resource Configuration](#extended-resource-configuration)
```

Comment:
I wonder if it would be best to have the `Extended Resource Configuration` as the second item after `Main Resource Configuration` and then `Module Configuration` as the last. This is from the perspective of configuring the module for the first time where Main and Extended will be related.

Reply:
that's the current state in the template: https://github.com/mineiros-io/terraform-module-template/blob/main/README.tfdoc.hcl#L160 so yes, i 100% agree ;)